### DISCUSSION TITLE
is it possible for the mask to be cropped?

### DISCUSSION THREAD

**Discussion by doodguymandood** [continue-revolution/sd-webui-segment-anything#72] [Q&A]
this would be amazing for processing images if it could crop the mask to be the same size as the square it's detected in. is that possible?

**Comment by continue-revolution** [comment]
It is technically very easy if a bounding box is available. It should not be too hard even if a bounding box is not available. However, I’m not sure what panel you want this feature. Batch panel, or single image panel, or both? You can give me more context on your use case to help me to decide which panel I shall add this feature.

**Comment by doodguymandood** [comment]
Ideally both, but batch preferable, whatever works for you. So in my case I both use it to crop out a person from a bunch of images and to single out people/guns/weapons/objects from a large character/item sheet. batch seems to detect every item you ask it to find and put it in one mask, but it would be awesome if it made a mask for each item detected, then cropped it
